<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dirc_scan_tab1">
     <attribute name="title">
      <string>目录扫描</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="url_lable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="url_lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>http://192.168.74.133</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start_scan_Button">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始扫描</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_scan_Button">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>停止扫描</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>网址程序</string>
      </property>
      <widget class="QCheckBox" name="asp_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ASP</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="php_checkBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PHP</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="sspx_checkBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ASPX</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jsp_checkBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>JSP</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jsp_checkBox_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>数据库</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jsp_checkBox_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>备份</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jsp_checkBox_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>高危</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>521</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>扫描状态</string>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>501</width>
         <height>16</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>扫描总数：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-----</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>已扫描：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-----</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-----</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-----</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="all_lineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="finished_lineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>线程数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="threads_lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="reuslt_tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>761</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Web指纹</string>
     </attribute>
     <widget class="QLineEdit" name="web_utl_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>http://192.168.74.135</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请输入网址:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="web_start_Button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="web_result_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>431</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>端口扫描</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>信息</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="port_ip_lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>192.168.74.134</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="source_ports_textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>331</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="port_scan_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始扫描</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="port_num_comboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOP50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOP100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOP1000</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>请先选择端口数量：</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>401</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>结果：</string>
      </property>
      <widget class="QTextEdit" name="result_port_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>域名解析</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>域名地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="domain_lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>baidu.com</string>
      </property>
     </widget>
     <widget class="QPushButton" name="domain_start_Button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始解析</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>241</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>A记录</string>
      </property>
      <widget class="QTextEdit" name="A_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>221</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>MX记录</string>
      </property>
      <widget class="QTextEdit" name="MX_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>NS记录</string>
      </property>
      <widget class="QTextEdit" name="NS_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>使用帮助</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>381</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;本程序有xxx小组制作。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>© hannoch</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能菜单</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>directory_scan()</slot>
 </slots>
</ui>
